src/window.rs

```diff
@@ -17,6 +18,7 @@ pub struct WindowConfig {
     pub(crate) transparent: Option<bool>,
     pub(crate) fullscreen: Option<Fullscreen>,
     pub(crate) window_icon: Option<bool>,
+    pub(crate) window_icon_value: Option<Icon>,
```

Comment:
I think we could change the previous line `window_icon` to be an Option<Icon> instead and remove `window_icon_value`.
This is a breaking change to the API - so the choice is for floem devs.

Reply:
I think we can re-use `window_icon` because it's not used anywhere

Reply:
This has been changed. It is much nicer this way, thank you for the agreement.